### ISSUE TITLE
Minumum Version for UX doesn't match UI

### ISSUE THREAD

**Issue by agavgavi** [Dorako/pf2e-dorako-ux#5]
Right now pf2e-dorako-ui has a minimum supported version of `10` with `pf2e` being `4.0.0`. Dorako-ux has a min of `11` and `pf2e` equal to `5.6.0`.

This means that UI can be used with the v10 version of foundry but it's dependency will not be able to load.

**Comment by Dorako** [comment]
That's true. I don't update the minimum versions at all. Running anything other than the latest version is always at your own risk.
The proper action is probably to raise the minimum this once, and refer v10 users to version earlier than 3.0
